dll/ntdll/def/ntdll.spec

```diff
@@ -7,9 +7,9 @@
 @ fastcall -arch=i386 RtlUlongByteSwap(long)
 @ fastcall -ret64 RtlUlonglongByteSwap(double)
 @ fastcall -arch=i386 RtlUshortByteSwap(long)
-@ stdcall -arch=i386,x86_64 ExpInterlockedPopEntrySListEnd()
-@ stdcall -arch=i386,x86_64 ExpInterlockedPopEntrySListFault()
-@ stdcall -arch=i386,x86_64 ExpInterlockedPopEntrySListResume()
+@ stdcall -arch=i386 ExpInterlockedPopEntrySListEnd()
```

Comment:
Why do we need them splitted?

Reply:
It was an attempt to get the order right, but in fact dlltool reorders them in the "wrong way". MS orders exports by symbol name, GCC/dlltool orders them somewhat by export name (in fact the logic is screwed, as it first does a qsort by export name, then a qsort that only sorts ordinal exports vs auto-ordinal). Proper solution would be to fix dlltool, workaround could be spec2def forcing ordinals, I'm looking into it...